File: Kotlin/프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.kt
class Solution {
    fun solution(food: IntArray): String {
        val sb = StringBuilder()

        for (i in 1 until food.size) {
            if (food[i] > 1) {
                for (j in 1 .. food[i] / 2) {
                    sb.append(i)
                }
            }
        }

        val rev = sb.reversed()



        return sb.append(0).append(rev).toString()
    }
}

File: C#/프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.cs
using System;

public class Solution {
    public string solution(int[] food) {
        string answer = "";
        for(int i = 1; i < food.Length; i++)
        {
            while(food[i] > 1)
            {
                answer += i.ToString();
                food[i] -= 2;
            }
        }
        
        answer += "0";
        var arr = answer.ToCharArray();
        for(int i = arr.Length - 2; i > -1; i--)
        {
            answer += arr[i];
        }
        
        return answer;
    }
}